docs/src/modules/components/DemoEditor.tsx

```diff
@@ -100,7 +104,7 @@ export default function DemoEditor(props: DemoEditorProps) {
       <div className="MuiCode-root" {...handlers}>
         <div className="scrollContainer">
           <StyledSimpleCodeEditor
-            padding={20}
+            padding={contextTheme.spacing(2)}
```

Comment:
To equal https://github.com/mui/material-ui/blob/cb278c426db9eb199283db50d058a3b28a871218/docs/src/modules/components/MarkdownElement.js#L20